core/modules/layout/css/layout.admin.css

```diff
@@ -395,3 +395,25 @@
 [dir="rtl"] .edit-layout-top .dropbutton-wrapper {
   float: left;
 }
+
+details summary {
+  list-style-type: none;
+}
+details[open] summary span::before, details summary span::before {
+  content: "";
+  position: relative;
+  left: -.3em;
+  top: -.2em;
+  width: 0;
+  height: 0;
```

Comment:
```
  width: 0;
  height: 0;
```
Are these two items necessary? I tried to disable them in the inspector and did not see any change.

Reply:
I thought they will help override any default browser styles. But not, will remove.